docs/sources/mimir/manage/tools/query-tee.md

```diff
@@ -87,14 +87,113 @@ A request sent from the query-tee to a backend includes HTTP basic authenticatio
+**YAML configuration example:**
+
+```yaml
+prometheus-hot:
+  request_headers:
+    X-Storage-Tier: ["hot"]
+  request_proportion: 1.0
+  min_time_threshold: "6h"
+
+prometheus-warm:
+  request_headers:
+    X-Storage-Tier: ["warm"]
+    Cache-Control: ["no-store"]
+  request_proportion: 0.8
+  min_time_threshold: "720h" # 30 days
+
+prometheus-cold:
+  request_headers:
+    X-Storage-Tier: ["cold"]
+  request_proportion: 0.5
+  min_time_threshold: "0s" # serves all queries
+```
+
 ### Backend selection
 
 You can use the query-tee to either send requests to all backends, or to send a proportion of requests to all backends and the remaining requests to only the preferred backend.
-You can configure this with the `-proxy.secondary-backends-request-proportion` CLI flag.
+
+#### Request proportion
+
+You can configure request proportions in two ways:
+
+1. **Global setting**: Use the `-proxy.secondary-backends-request-proportion` CLI flag to set the default proportion for all secondary backends.
+2. **Per-backend setting**: Use the `request_proportion` field in the backend configuration file to override the global setting for individual backends.
 
 For example, if you set the `-proxy.secondary-backends-request-proportion` CLI flag to `1.0`, then all requests are sent to all backends.
 Alternatively, if you set the `-proxy.secondary-backends-request-proportion` CLI flag to `0.2`, then 20% of requests are sent to all backends, and the remaining 80% of requests are sent only to your preferred backend.
 
+Per-backend request proportions take precedence over the global setting. In the configuration example above, `prometheus-warm` would receive 80% of requests and `prometheus-cold` would receive 50% of requests, regardless of the global setting.
+
+#### Time-based routing
+
+You can configure backends to only serve queries based on the time range of the requested data using the `min_time_threshold` setting. This is useful for implementing tiered storage architectures where different backends store data for different time periods.
+
+**How time-based routing works:**
+
+- A backend with `min_time_threshold: "24h"` will only serve queries where the minimum query time is within the last 24 hours
+- A backend with `min_time_threshold: "0s"` (the default) will serve all queries regardless of their time range
+- The **preferred backend is always included** regardless of its time threshold
+- For range queries (`/api/v1/query_range`), the earliest time between `start` and `end` parameters is used
+- For instant queries (`/api/v1/query`), the `time` parameter is used, or current time if not specified
+
+**Example scenario:**
+
+With the configuration example above:
+
+- Recent queries (< 6 hours old) → sent to `prometheus-hot`, `prometheus-warm`, and `prometheus-cold`
+- Medium queries (6h to 30 days old) → sent to `prometheus-warm` and `prometheus-cold` only
+- Old queries (> 30 days old) → sent to `prometheus-cold` only (plus preferred backend if different)
```

Comment:
There's something I don't quite get from the examples, I think there might be some typos or I am not following the logic. For example wouldn't `(< 6 hours old)` go to `prometheus-cold` only? 

AFAIU we only discard a request for a backend if it's too recent, not if it's too old. I think one thing that might be causing confusion is `prometheus-cold` is configured to serve all, while `prometheus-hot` serves only 6h-old data.